hiera.yaml

```diff
@@ -10,7 +10,7 @@
     - common
 
 :yaml:
-    :datadir: /opt/puppet/env/%{environment}/hieradata
+    :datadir: /etc/puppetlabs/code/environments/%{environment}/hieradata
```

Comment:
Same, it's really long, but there is a symlink still in place if you want to edit things at the old location